src/Uno.UI.Tests/Windows_Globalization/When_Calendar.cs

```diff
@@ -19,8 +19,7 @@ public void When_DateTimeOffset_Is_Next_Day_If_Converted_To_Utc()
 			var calendar = new WG.Calendar();
 			var offset = new DateTimeOffset(year: 2023, month: 5, day: 1, hour: 21, minute: 0, second: 0, TimeSpan.FromHours(-5));
 			calendar.SetDateTime(offset);
-			// calendar.Day must be 1 here to match WinUI, **and also** not break CalendarView selection of "Today"
-			Assert.AreEqual(1, calendar.Day);
+			Assert.AreEqual(2, calendar.Day);
```

Comment:
I confirmed this is `2` on WinUI. Not sure how I initially thought it must be 1 😕